packages/notebook/src/browser/notebook-editor-widget.tsx

```diff
@@ -29,6 +29,8 @@ import { Emitter } from '@theia/core/shared/vscode-languageserver-protocol';
 import { NotebookEditorWidgetService } from './service/notebook-editor-widget-service';
 import { NotebookMainToolbarRenderer } from './view/notebook-main-toolbar';
 import { Deferred } from '@theia/core/lib/common/promise-util';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Minor: This doesn't seem to be actually necessary. Your eslint vscode extension probably just didn't recognize it, when you installed the respective package.